index.html

```diff
@@ -12,14 +12,19 @@
         <header>
             <a class="logo" href="#">ABU TALHA</a>
             <nav class="menu-icon">
-                <img class="menubars" src="images/menu.png" alt="Menu"> 
+                <img class="menubars" src="images/menu.png" alt="Menu">
+                <ul class="heading-menu">
+                  <li>Portfolio</li>
+                  <li>About</li>
+                  <li>Contact</li>
+              </ul> 
             </nav>
         </header>
         <section class="homepage">
             <div class="intro">
                 <h1>
                     I'm Abu Talha. <br>  
-                    Glad to see <br> you!
+                    Glad to see <br class="break"> you!
                 </h1>                                  
             </div>
```

Comment:
Kindly adjust the hero section to match the design. The items below are vertically aligned to the left.

Your design:
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/20554583/866872df-637e-4636-9de6-e2757fd70216)


Figma design:
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/20554583/563817f3-4509-4570-974c-60ed9f3c63db)